changes.go

```diff
@@ -32,6 +35,9 @@ func (d *db) Changes(ctx context.Context, opts map[string]interface{}) (driver.C
 	if err = chttp.ResponseError(resp); err != nil {
 		return nil, err
 	}
+	if overrideOpts == nil {
+		return newChangesNormal(resp.Body), nil
```

Comment:
What is the reason for a separate implementation for the normal feed? The format should be the same, no?

Reply:
I think this makes sense now. In 'continuous' mode, the stream is just a stream of JSON objects. In other modes, it's a single JSON object.